Quick Paste/ViewController.swift

```diff
@@ -348,17 +371,11 @@ extension ViewController: NSFetchedResultsControllerDelegate {
         tableView.endUpdates()
         //deallocate consolidator
         consolidator = nil
-        print("tableView select row before manual set: \(tableView.selectedRow)")
-        print("rowToBeFocusedIndex: \(rowToBeFocusedIndex)")
-        //if rowToBeFocus is -1, rowToBeFocusIndex would be nil, that would lead to stmt below unwrap nil(crash)
-        if rowToBeFocusedIndex != nil{
-            tableView.selectRowIndexes(rowToBeFocusedIndex as IndexSet, byExtendingSelection: false)
-        }
```

Comment:
manually set rowToBeFocus should not apply to insertion. Keep them on same index would end up with unintended focus